inspec.yml

```diff
@@ -18,11 +18,35 @@ supports:
 
 # The following defines the default inputs for the configurable controls used in the RHEL 7 DISA STIG.
 inputs:
+  # SV-204393, SV-204394, SV-204395, SV-204397, SV-204426, SV-204431, SV-204447, SV-204448, SV-204578, SV-204580, SV-204595, SV-204603, SV-204631, SV-204632, SV-204633
+  - name: org_name
+    desc: "Name of the organization running this profile"
+    type: Hash
+    value: "DoD"
```

Comment:
The value is currently set to `String` so it doesn't match the type `Hash`. I think we usually use 2 keys for the acronym and full name